Fix #10662: Fixed font issue on create/remove ducks tooltip (#10861)
OpenRCT2/OpenRCT2: data/language/en-GB.txt

STR_6353    :{SMALLFONT}{BLACK}Low density
STR_6354    :{SMALLFONT}{BLACK}Medium density
STR_6355    :{SMALLFONT}{BLACK}High density
STR_6356    :{SMALLFONT}{BLACK}Spawns ducks if park contains water

> STR_6357    :{SMALLFONT}{BLACK}Removes ducks if park contains ducks

Reviewer: This seems a bit redundant to me. I would change it to this:

```suggestion
STR_6357    :{SMALLFONT}{BLACK}Removes all ducks from the map
```

Reviewer 2: Indeed, that seems like a better tooltip.
